gemrb/core/Scriptable/Actor.cpp

```diff
@@ -878,12 +878,14 @@ static void pcf_morale (Actor *actor, ieDword /*oldValue*/, ieDword /*newValue*/
 		// recover from panic, since morale has risen again
 		// but only if we have really just recovered, so panic from other
 		// sources isn't affected
-		if ((actor->Modified[IE_MORALE]-1 == actor->Modified[IE_MORALEBREAK]) || (actor->Modified[IE_MORALEBREAK] == 0) ) {
+		if ((actor->Modified[IE_MORALE]-1 >= actor->Modified[IE_MORALEBREAK]) || (actor->Modified[IE_MORALEBREAK] == 0) ) {
 			if (!third || !(actor->Modified[IE_SPECFLAGS]&SPECF_DRIVEN)) {
 				actor->SetBaseBit(IE_STATE_ID, STATE_PANIC, false);
+				actor->BaseStats[IE_CHECKFORBERSERK] = 0;
```

Comment:
Eh, not sure ... `SetCircleSize` actually looks at the `Modified` one. Can I simply set this here?

Reply:
It's fine, matches the rest. The other one checks Modified, since also effects can set the stat.